requirements.txt

```diff
@@ -1,2 +1,3 @@
-PyYAML
-websocket-client
\ No newline at end of file
+PyYAML==6.0
```

Comment:
Note that we also need to update the `install_requires` list in `setup.py` (which is what `pip` actually uses when someone installs `ops`). See https://github.com/canonical/operator/pull/891/commits/903c9f08b80345d99a6e1b57b9f17f5eea6fe7dc

Reply:
I'll address this.

My bad, I didn't notice a `setup.py` file in the project tree. (I saw the `pyproject.toml` and thought that would be it - do you need a `pyproject.toml` and `setup.py` in a project?).

Reply:
Do you need both? I thought `setup.py` was for pip and `pyproject.toml` just for tools ... but it looks like I was behind the times! For example https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/

Good to know. In any case, I don't mind considering that migrating to just use `pyproject.toml`, but no need to consider that in this PR.

Reply:
Yeah, I learnt about it recently too: https://github.com/orgs/stdize/discussions/1

Reply:
Tracking this as an item in https://github.com/canonical/operator/issues/893